docs/source/usage.rst

```diff
@@ -0,0 +1,186 @@
+Using cool-seq-tool
```

Comment:
I'd really appreciate feedback about the wording and accuracy of the description for this page -- I felt like I might be off base about how certain parameters are used